docs/rules/no-multi-spaces.md

```diff
@@ -25,6 +34,36 @@ Examples of **correct** code for this rule:
 />
 ```
 
-### Options
+```html
+<i
+  :class="{
+    'fa-angle-up' : isExpanded,
+    'fa-angle-down' : !isExpanded,
+  }"
+/>
+```
+
+## :wrench: Options
+
+This rule has an object option:
 
-Nothing
+`"ignoreProperties": false` (default) whether or not objects' properties should be ignored
+
+### Example:
+
+```json
+"vue/no-multi-spaces": [2, {
+  "ignoreProperties": true
+}]
+```
+
+:+1: Examples of **correct** code for this rule:
+
+```html
+<i
+  :class="{
+    'fa-angle-up'   : isExpanded,
+    'fa-angle-down' : !isExpanded,
```

Comment:
@michalsnik Hello!
Is the spaces after the colon allowed?

```js

    'fa-angle-up':   isExpanded,
    'fa-angle-down': !isExpanded,
```

It seems that this case is allowed by option `{ "Property": true }` of core [no-multi-spaces](https://eslint.org/docs/rules/no-multi-spaces) rule,
so I think that it will be the same behavior is good.

Reply:
Good catch! I updated the cases and handle this one as well.